docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,115 @@
+* **You want a tool that allows you to manage Node.js versions as well as other dependencies.** Having a single-use tool like nvm is fine, but what if you could install node, PostgreSQL, nginx, etc. with a single command? Flox let's you do just that, and it also [allows you to configure and run services][services]{:target="\_blank"} like PostgreSQL simply by adding the `--start-services` argument to `flox activate`.
+* **You need to manage versions of other JavaScript runtimes, like Bun or Deno.** While nvm is, as the name denotes, used for managing Node.js versions, that's all it does. You can use Flox to install node, but you can also use it to install different JavaScript runtimes, including [Bun](https://bun.sh/){:target="\_blank"} and [Deno](https://deno.com/){:target="\_blank"}.
+
+## Install Flox
+Download and install Flox as described in our [installation instructions][install_flox]{:target="\_blank"}.
+
+## Create a Flox environment in your existing project and install Node.js
+Navigate to your project's directory and run `flox init`, and then search for the version of Node.js that is currently included in your `.nvmrc`, or in the `"engines"` property of your `package.json`. (Note that node is listed as `nodejs` and variants thereof in the Flox catalog.)
+
+```zsh
+➜  node-project git:(main) ✗ flox search nodejs
+nodejs       Event-driven I/O framework for the V8 JavaScript engine
+nodejs_14    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_16    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_18    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_19    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_20    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_21    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_22    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_23    Event-driven I/O framework for the V8 JavaScript engine
+nodejs-10_x  Event-driven I/O framework for the V8 JavaScript engine
+
+Showing 10 of 67 results. Use `flox search nodejs --all` to see the full list.
+
+Use 'flox show <package>' to see available versions
+```
+
+There is _a lot_ of software in the Flox Catalog, so you should have no trouble finding the node version you need. For this project, perhaps you decide to install Node.js v22.10.0.
+
+```zsh
+➜  node-project git:(main) ✗ flox install nodejs_22
+✅ 'nodejs' installed to environment 'node-project'
+```
```

Comment:
Any reason why this wouldn't say `'nodejs_22' installed to environment` instead of `nodejs`?

Reply:
Upon some reflection and discussion, I think we should advocate for users to install specific versions of node rather than the generic `nodejs` package, which to me seems like an alias for "latest" or something to that effect.